.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "airbnb",
+  "rules": {
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "linebreak-style": 0,
+    "react/prop-types": [2, { "ignore": ["children"] }],
```

Comment:
It's generally a bad idea to remove `children` as a prop-type.

Reply:
I'm not sure how to validate the children. I've removed the rule and validated children as PropTypes.node.

Please advise if this is right?

Reply:
Yeah, so **node** basically means anything that can be written in JSX.